docs/swagger.yaml

```diff
@@ -68,6 +68,11 @@ paths:
           in: query
           description: Filter by directProjectId
           type: integer
+        - name: type
+          in: query
+          description: Filter by type abbreviation, exact match. If provided, the typeId will be ignored
```

Comment:
I like this. Add something similar to the description of typeId. "If type is provided, typeId will be ignored". People don't read up :)